tests/test_ell1h.py

```diff
@@ -143,6 +145,20 @@ def test_J0613_STIG(toasJ0613, modelJ0613_STIG):
     f.fit_toas()
 
 
+def test_SINI_error():
+    """Test SINI and M2 error.
+    """
+    SINI_par = simple_par.replace("H3 2.7507208E-7", "SINI 0.8")
+    with pytest.raises(ValueError, match="'SINI' will not be used in ELL1H model. "):
+        model.get_model(StringIO(SINI_par))
+
+
+def test_M2_warning():
```

Comment:
Please change this function name as well, for exactly the same reason as the call above.